cli/tools/lint/mod.rs

```diff
@@ -218,7 +220,7 @@ async fn lint_files(
 
   futures.push({
     let has_error = has_error.clone();
-    let linter = create_linter(lint_rules.rules);
+    let linter = linter.clone();
```

Comment:
@bartlomieju This might works? creating an instance outside the future and cloning it when it is used inside the future.